client/src/util.ts

```diff
@@ -2,29 +2,66 @@ import { DIDDocument, PublicKey } from 'did-resolver';
 import { Account, PublicKey as SolanaPublicKey } from '@solana/web3.js';
 import { DID_METHOD } from './constants';
 import { ClusterType } from './solid-data';
+import { decode, encode } from 'bs58';
 
 // a 64-byte private key on the X25519 curve.
 // In string form it is base58-encoded
-export type PrivateKey = number[] | string | Buffer;
+export type PrivateKey = number[] | string | Buffer | Uint8Array;
```

Comment:
I've been wondering about this type -- is it meant for when people want to sign Solana transactions using their private key, no matter what format it's in?

Reply:
Yes exactly. e.g. it allows you to pass in a base58 encoded string, which is easier for curl calls or command-line tools.